CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_linkedin_source v0.7.0
+## 🚨 Breaking Changes 🚨
+Due to Linkedin Ads API [change in January 2023](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/recent-changes?view=li-lms-2022-12#january-2023), there have been updates in the Linkedin Ads Fivetran Connector and therefore, updates to this Linkedin package. 
+
+The following fields have been completely deprecated in the `stg_linkedin_ads__creative_history` model ([PR #48](https://github.com/fivetran/dbt_linkedin_source/pull/48)):
+- `type`
+- `call_to_action_label_type`
+- `version_tag`
+
+## Updates
+- The following legacy fields have been updated respectively in the connector and [PR #48](https://github.com/fivetran/ dbt_linkedin_source/pull/48) includes the below modifications:
+  - `last_modified_time` has been updated to `last_modified_at`
+  - `created_time` has been updated to `created_at`
+  - `status` has been updated to `intended_status`
+- `src_linkedin.yml` have been updated to reflect new definitions for the above updated fields.
+
+## Under the Hood
+- `integration_tests/seeds/linkedin_creative_history_data` has been updated to reflect new fields and deprecated fields
+- Uniqueness testing in `stg_linkedin_ads__creative_history` has been updated to now include `_fivetran_synced` and `last_modified_at` in place of the previous test with `version_tag`. 
```

Comment:
Same comment from the staging model about the inclusion of _fivetran_synced being included in the test. I would prefer we remove it if you think that is a possibility.

Reply:
Removed!